apps/web/src/components/ui/sheet.tsx

```
@@ -74,10 +74,7 @@ const SheetContent = (props: SheetContentProps) => {
 
   return (
     <Portal mount={document.body}>
-      <div
-        class="fixed inset-0 z-[--z-modal] flex"
-        onClick={() => local.onClose?.()}
-      >
+      <div class="fixed inset-0 z-[--z-modal] flex" onClick={() => local.onClose?.()}>
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)